consensus/istanbul/core/handler.go

```diff
@@ -193,7 +199,26 @@ func (c *core) handleCheckedMsg(msg *message, src istanbul.Validator) error {
 	return errInvalidMessage
 }
 
-func (c *core) handleTimeoutMsg() {
+func (c *core) handleTimeoutMsg(view *istanbul.View) {
+	// TODO-Klaytn-Istanbul: Check whether this statement is required. EN/PN should not handle consensus msgs.
+	if c.backend.NodeType() != common.CONSENSUSNODE {
```

Comment:
@aidan-kwon 
Please check if this log is printed in EN/PN and remove this if it is not printed.

Reply:
EN/PN can reach the log.